web/client/utils/FileUtils.js

```diff
@@ -9,9 +9,18 @@
 const FileSaver = require('file-saver');
 const toBlob = require('canvas-to-blob');
 const shp = require('shpjs');
+const tj = require('@mapbox/togeojson');
+const JSZip = require('jszip');
 const {Promise} = require('es6-promise');
+const parser = new DOMParser();
 
 const FileUtils = {
+    MIME_LOOKUPS: {
+        'gpx': 'application/gpx+xml',
+        'kmz': 'application/vnd.google-earth.kmz',
+        'kml': 'application/vnd.google-earth.kml+xml',
+        'zip': ['application/x-zip-compressed', 'application/zip']
```

Comment:
Where do you handle MIME_LOOKUPS['zip'] being an array? It seems to me that MIME_LOOKUPS['zip'] is expected to be a string

Reply:
I'm changing it